pkgs/applications/networking/mailreaders/meli/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, pkgconfig
+, openssl
+, dbus
+, sqlite
+, file
+, gzip
+, notmuch
+, withNotmuch ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "meli";
+  version = "alpha-0.5.1";
+
+  src = fetchgit {
+    url = "https://git.meli.delivery/meli/meli.git";
+    rev = "${version}";
+    sha256 = "sha256:1y5567hdm1s2s272drxvmp6x4y1jpyl7423iz58hgqcsjm9085zv";
+  };
+
+  cargoSha256 = "sha256:040dfr09bg5z5pn68dy323hcppd599d3f6k7zxqw5f8n4whnlc9y";
```

Comment:
```suggestion
  cargoSha256 = "040dfr09bg5z5pn68dy323hcppd599d3f6k7zxqw5f8n4whnlc9y";
```

Reply:
I already count more than 300 instances of SRI hashes in Nixpkgs `master` so I thought it would be safe to start using them. Could you share your rationale?

Reply:
AFAIK it's not current best practice. At the very least, it's extra noise/repetition with the name already being cargoSha256.

Reply:
Okay, fixed it.